### ISSUE TITLE
Error downloading current season data, schedule

### ISSUE THREAD

**Issue by EricPPeterson** [sportsdataverse/hoopR#130]
When I try and download the 2023-2024 schedule, pbp data and player box scores, I get an error:

x <- load_nba_schedule(seasons = most_recent_nba_season())
Warning messages:
1: In readRDS(con) :
  cannot open URL 'https://raw.githubusercontent.com/sportsdataverse/hoopR-data/main/nba/schedules/rds/nba_schedule_2024.rds': HTTP status was '404 Not Found'
2: Failed to readRDS from <https://raw.githubusercontentI .com/sportsdataverse/hoopR-data/main/nba/schedules/rds/nba_schedule_2024.rds>

I get the same issue when trying to download anything from the 2023-2024 season. Has this data just not been uploaded yet? 

thanks ..

Eric

**Comment by Coop197823** [comment]
similar issue to mine that has been [closed](https://github.com/sportsdataverse/hoopR/issues/128).  I had to update the package and it worked fine

**Comment by aggieed97** [comment]
@Coop197823 Yes, I had seen that your issue had been closed, but the link that the data is supposed to be coming from doesn't exist.  I have v2.0.1 as well, and it still doesn't work.

<img width="1114" alt="image" src="https://github.com/sportsdataverse/hoopR/assets/35159749/6bda0f02-9985-400d-82b3-955165991e7b">

**Comment by Coop197823** [comment]
I see it here at this link, and this is the directory you should be pointed to (or well at least I am) https://github.com/sportsdataverse/sportsdataverse-data/releases/tag/espn_nba_schedules

also try to specify **hoopR**:: in ` hoopR::load_nba_schedule(seasons = most_recent_nba_season()) `

**Comment by aggieed97** [comment]
Womp, womp.  You are correct.  NBA scheds are in but it looks like the NCAA Men's Basketball sched is not yet updated for 2023-2024.  I saw that you already opened up another issue for that a few days ago.

**Comment by Coop197823** [comment]
@aggieed97  you should respond to my issue to see if we can get an expedited solution

**Comment by saiemgilani** [comment]
```r
remotes::install_github("sportsdataverse/hoopR")
```

And try again.

**Comment by saiemgilani** [comment]
The 2023-era current-season download failures are long resolved — the daily producers now publish schedules/box/pbp current through each season's end (verified across the 2024-2026 assets while triaging the newer staleness reports). Closing; please open a fresh issue if anything specific still fails on 3.1.0.
